src/20ec0_len_5040.c

```diff
@@ -1,5 +1,60 @@
 #include "common.h"
 #include "npc.h"
+#include "hud_element.h"
+
+typedef struct HudComplexStatusIcon {
+	/* 0x0 */ s8 active;
+	/* 0x1 */ s8 removing;
+	/* 0x2 */ s8 unk_02;
+	/* 0x3 */ s8 unk_03;
+	/* 0x4 */ s8 unk_04;
+	/* 0x5 */ s8 unk_05;
+	/* 0x8 */ s32 hudElemIndex;
+	/* 0xC */ s32 unk_0C;
+} HudComplexStatusIcon;/* size = 0x10 */
```

Comment:
Doesn't matter too much but we've gone for this style of comment so far
```suggestion
} HudComplexStatusIcon; // size = 0x10
```